SPECS/python-gast/python-gast.spec

```diff
@@ -0,0 +1,112 @@
+%global _description %{expand:
```

Comment:
Just wondering if whomever requires this package can forcibly build with a python3 version...since we don't have python2 in Mariner 2.0.  

Ok to add it, but just wondering if we have a way out of it.

Reply:
I did remove all the python2 code from the spec. I might remove Python2 from this description as well.